src/data/messaging.js

```diff
@@ -25,52 +26,44 @@ function init() {
 async function loadConversations() {
   try {
     const myConversations = await getMyConversations()
-
+    
     conversations.set(myConversations)
   } catch (e) {
-    console.error(`messaging.js:loadConversations: `, e)
-    // TODO: errorhandling?
+    clearInterval(intervalId)
+
+    throw e
   }
 }
 
 export async function saw(conversationId) {
-  try {
-    const updatedConversation = await markMessagesAsRead(conversationId)
+  const updatedConversation = await markMessagesAsRead(conversationId)
 
-    unreads.update(currentUnreads => {
-      const matchingUnread = currentUnreads.find(unread => unread.id === updatedConversation.id)
-      
-      if (matchingUnread) {
-        // instead of simply matchingUnread.count = 0, leave room for the possibility 
-        // another message came in during this update, i.e., maybe it's not a 0
-        matchingUnread.count = updatedConversation.unreadMessageCount
-      }
-      
-      return currentUnreads.filter(({ count }) => count > 0)
-    })
-  } catch (e) {
```

Comment:
We might want to add this back, in order to silently "swallow" this error since there's probably nothing the user can do about it.